components/Main/Main.js

```diff
@@ -47,31 +50,35 @@ const Main = () => {
   const button = Button({
     textContent: 'Refresh',
     className: 'button',
+    onclick: () => {
+      (async () => {
+        const { result } = await Api.get();
+        UnorderList.textContent = '';
+        result.forEach((eachList, index) => {
+          const list = List({
+            className: 'player',
+            id: `playerID-${index + 1}`,
+          });
+
+          const paragraph = Paragraph({
+            className: 'player_detail',
+            textContent: `${eachList.user}: `,
+          });
+
+          const span = ChildText({
+            className: 'score',
+            textContent: eachList.score,
+          });
+
+          paragraph.append(span);
+          list.append(paragraph);
```

Comment:
- [ ] It would be nice to have a list of the leaderboard scores as soon as the page loads. Currently, you can only view the leaderboard scores when you click the refresh button.
![image](https://user-images.githubusercontent.com/46839548/186992882-79340bea-4a87-4630-b1d5-27a1f3513be2.png)

Reply:
Thank you for pointing out issue @codepantha 👍🏻